modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ModelBuilderContext/objectFolderReducer.ts

```diff
@@ -613,47 +563,6 @@ export function ObjectFolderReducer(state: TState, action: TAction): TState {
 			};
 		}
 
-		case TYPES.DELETE_OBJECT_DEFINITION: {
```

Comment:
Hi @naraandrade. I've removed **DELETE_OBJECT_DEFINITION** case because we are not using it anymore. Previously it was being used on the action "move to current folder" , where we added the definition on the new folder and removed from the old folder. But we decided to change this approach to use **UPDATE_MODEL_BUILDER_STRUCTURE** since the back end already returns the correct structure of folder after the move action.

Reply:
Oh, I see. Thanks for the explanation, @alinedoleron!